stl/inc/functional

```diff
@@ -1340,7 +1352,11 @@ _NODISCARD void* _Function_new_large(_CTypes&&... _Args) {
     if constexpr (alignof(_Vt) <= __STDCPP_DEFAULT_NEW_ALIGNMENT__) {
         _Ptr = ::operator new(sizeof(_Vt));
     } else {
+#ifdef __cpp_aligned_new
         _Ptr = ::operator new (sizeof(_Vt), align_val_t{alignof(_Vt)});
+#else // ^^^ __cpp_aligned_new / !__cpp_aligned_new vvv
+        _Ptr = _aligned_malloc(sizeof(_Vt), alignof(_Vt));
+#endif // __cpp_aligned_new

```

Comment:
The `::operator new()` calls are throwing, so the `_aligned_malloc()` call needs to detect allocation failure and call `_Xbad_alloc()`.